go/src/simple_pub/simple_pub.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"dps"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	// Pre-shared keys for testing only. DO NOT USE THESE KEYS IN A REAL APPLICATION!
+	networkKeyId := dps.KeyId{
```

Comment:
Can the preshared keys and certs be put in a package than can be shared by the samples apps?